proto/ibc/core/client/v1/query.proto

```diff
@@ -36,6 +36,11 @@ service Query {
     option (google.api.http).get = "/ibc/core/client/v1/consensus_states/{client_id}";
   }
 
+  // ConsensusStateHeights queries the height of every consensus states associated with a given client.
+  rpc ConsensusStateHeights(QueryConsensusStateHeightsRequest) returns (QueryConsensusStateHeightsResponse) {
+    option (google.api.http).get = "/ibc/core/client/v1/consensus_states/{client_id}/heights";
```

Comment:
The original PR had the route `"/ibc/core/client/v1/consensus_states/heights/{client_id}"`, but I just realized that it would be more consistent with `heights` at the end, so I changed it here. It's a bit sneaky, but I hope it's ok. I will also open a PR to main to update the route.